### DISCUSSION TITLE
storjlabs watchtower on Raspbian 10 / Raspberry Pi 4

### DISCUSSION THREAD

**Discussion by hostingnuggets** [containrrr/watchtower#975] [Q&A] [answered]
Hello,

I am running the watchtower fork from storjlabs like this on my Raspbian 10 (Raspberry Pi 4):

```
docker run -d --restart=always --name watchtower -v /var/run/docker.sock:/var/run/docker.sock storjlabs/watchtower storagenode watchtower --stop-timeout 300s
```

and get the following warning:

```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm/v7) and no specific platform was requested
```

still if I check with `docker ps` I can see the container running. But I was wondering if it really works, or am I doing something wrong here?

**Comment by piksel** [comment, marked as answer]
Why are you asking here when using a fork? They are probably building a Docker image without setting the base image platform, which defaults it to the builder's platform.

**Reply by hostingnuggets** [reply]
Thank you for your answer. So it means it is working anyway and I can ignore the warning?

I am asking here because on the storj watchtower github page I could not find the issues tab, so I was not able to report the issue there.

**Reply by piksel** [reply]
Maybe. I don't know what their image contains. It's over a year old though, so even if it's built from an unmodified clone of our repo we can't really support it.
The warning itself doesn't tell you whether it's working or not, just that the image was tagged for another platform.

**Reply by hostingnuggets** [reply]
Thank you for your clear answer.
